src/PublicShareAuthSidebar.vue

```diff
@@ -187,6 +174,36 @@ export default {
 				this.$store.dispatch('updateToken', '')
 			}
 		},
+
+		async postJoinConversation() {
+			// Fetching the conversation needs to be done once the user has
+			// joined the conversation (otherwise only limited data would be
+			// received if the user was not a participant of the conversation
+			// yet).
+			await this.fetchCurrentConversation()
+
+			// Joining the call needs to be done once the participant identifier
+			// has been set, which is done once the conversation has been
+			// fetched. MediaSettings are called to set up audio and video devices
+			// and also to give a consent to recording, if set up
+			emit('talk:media-settings:show')
```

Comment:
everything apart from this line should be safe to keep in `joinConversation`, as it's only about fetching the conversation, that shouldn't be blocking for unknown guests (to let them know a moderator has started a call, for example)